### PR TITLE
added transcriptons

### PR DESCRIPTION
**Pull request by spapando**

### CODE REVIEW

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml [hunk 1]
```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
...
+                <lb n="26"/>δυσιμερίας π<supplied reason="omitted">λ</supplied>ήσατη τοῦ αὐτοῦ
+                <lb n="27"/><name key="Βαβύλας Babylas">Βαβυλᾶ</name> λαχανωπώλι ὥν ἤτηκ
+                <lb n="28" break="no"/>εν <name key="Εἱσυχία Hesykhia">Εἱσυχία</name>.
+            </ab></div>
             
             <div type="translation">
-            <p></p></div>
+            <p>
+                <hi rend="italic">Britakulma arôdasidaki[.]a 
+                    (2) êllim abri makhrêmella astrak iaêou sathnanô ôpakh
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml:212]: We'll have to see how this renders once committed to main; I think if we want to keep the lines as they are here we might need to put them in their own `<p>` elements, but we can revisit that after we see how it looks.

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml [hunk 2]
```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
...
+                <lb n="3"/>αστρακ ιαηου σαθνανω ωπαχ
+                <lb n="4"/>θαλσου σανταρμα χουχι
+                <lb n="5"/>ερηβαθ σαρδοασιρως αχωρ<add place="above">ω</add>
+                <lb n="6"/>ὦ βρώ<supplied reason="omitted">ν</supplied>των καὶ ἀστ<supplied reason="omitted">ρ</supplied>άπτων <name type="divine" key="Iao Ιαω">Ιαω</name>
+                <lb n="7"/>βάλη δῆσων σύ<supplied reason="omitted">ν</supplied>δησω<supplied reason="omitted">ν</supplied> <name key="Βαβύλας Babylas">Βαβυλᾶν</name>
+                <lb n="8"/>τὼν λαχανωπώλην ὥ<supplied reason="omitted">ν</supplied> ἤτηκεν
+                <lb n="9"/>εἱ μιηρὰ μέτρα <name key="Διωνυσία Dionysia">Διωνυσία</name> εἱ κὲ
+                <lb n="10"/><name key="Εἱσυχία Hesykhia">Εἱσυχία</name> ὐκῶ<supplied reason="omitted">ντα</supplied> ἐν γιτωνία Μυγδ
+                <lb n="11" break="no"/>ωνιτῶν: ὡς ἔβαλες τὼ ἅ<supplied reason="omitted">ρ</supplied>μα το
+                <lb n="12" break="no"/>ῦ Φαραῶνος, οὕτος βάλη τὶν δύσ
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml:190]: Maybe worth putting some name markup around Φαραῶνος here, kind of like how we do with Caesar when it's a title?

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml [hunk 3]
```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
...
+                <lb n="13" break="no"/><surplus>ω</surplus>ληψιν αὐτοῦ: ὦ βρ<supplied reason="omitted">ό</supplied>ντων καὶ
+                <lb n="14"/>ἀσ<surplus>σ</surplus>τρ<supplied reason="omitted">ά</supplied>πτων <name type="divine" key="Iao Ιαω">Ιαω</name>, ὡς ἐξέκωψ
+                <lb n="15" break="no"/>ης τὰ πρωτότυκα τῖς Ἠγύπ
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml:193]: Consider adding a quick `<placeName>` for Egypt here?

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml [hunk 4]
```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
...
+                <lb n="16" break="no"/>του, ἄ<supplied reason="omitted">κ</supplied>κωψων τ<unclear>ἄλο</unclear><supplied reason="lost">γα α</supplied> 
+                <lb n="17" break="no"/>ὐτο<supplied reason="omitted">ῦ</supplied> τόσυν <surplus>ΟΣΤ</surplus> ὅσ<surplus>ω</surplus>περ Η<unclear> </unclear><gap reason="lost" quantity="3" unit="character"/>
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml:195]: I think the H in this line is meant to be inside the unclear?

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130b.xml [hunk 5]
```diff
@@ -172,10 +178,32 @@ changes: Added @evidence to date
     <text>
         <body>
             <div type="edition">
-                <ab></ab></div>
+                <ab>
+                    <lb n="1"/>ὡ<unclear>ρ</unclear>χίζω ὑμᾶ<supplied reason="omitted">ς</supplied> διὰ τὼν
+                    <lb n="2"/>ἄνωμων καὶ δυσεβὴν
+                    <lb n="3"/><name key="Βαβύλας Babylas">Βαβυλᾶν</name> τὼν λαχανω
+                    <lb n="4" break="no"/>πώλιν: ὥσ<surplus>υ</surplus>πηρ ὑμῖς
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130b.xml:185]: Colon or ano teleia?

**Review by sdigiulio — APPROVED** [review]
Looks good; see minor queries for enhancements. Also, based on poking around on the Princeton museum website, I think the museum refs for the objects might have changed? (e.g. https://artmuseum.princeton.edu/collections/objects/39569 for Babylas curse tablet is listed is 2011-145 a-c.) If so we might need to revisit US ids, or at least get the new numbers in as altIdentifiers.

File: xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130b.xml (hunk 5 above)
Comment (spapando) [reply xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130b.xml:185]: ano teleia I think; how should I encode that?

**Pull request merged**